src/magento-payments/payouts.md

```diff
@@ -0,0 +1,128 @@
+---
+ee_only: false
+title: Payouts
+---
+
+Payouts shows comprehensive payout information at-a-glance, allowing you full transparency into the processed volume, payment balance, and detailed reporting on the transaction level for financial reconciliation.
+
+You don't have to open multiple dashboards or views to cross-reference orders and payments or reconcile accounts. Magento Payments enables you to take all these actions from one place---Payouts---so that you can view and manage your payouts efficiently.
```

Comment:
Having these views consolidated in one view can allow the user to be more efficient at managing payout but effective at understanding what is going on. 

This is just a thought: 

Orders, payments and reconciliations are represented in a single view for efficient cross referencing from one place---Paymouts---. This enables you to be conversant and manage payouts.